### ISSUE TITLE
Stop bundling shitting software anymore

### ISSUE THREAD

**Issue by IntPtrZero** [cheat-engine/cheat-engine#2807]
If you guys need money, you can simply attach a QR code for donate, sell this program on Steam or anything else. I really feel ashamed for you all. It's so fucking low to download rubbish on users' PC. So fucking LOW!

**Comment by IntPtrZero** [comment]
OK, you guys really did a fucking "GOOD" job. After google for the solution, what i found really shocked me. Even if you get the "Clean" version of the CE installer and you installed it, then you'll get a fucking big surprise: they added some lua scripts into your CE dir, which will cause your PC playing the creepy laughting sound and random resize the CE window. What's more, the "Clean" installer will only install x86 version for you. Alought some nice guys telled how to remove the scripts and how to fully installing CE, i decide don't to use this bullshit anymore. I can't put up with these fucking gimmicks, just like a fucking fly in my bowl, i could pick it out and keep eating as if nothing happened. But we all know the meal is fucking polluted. So that's it, i know some guys don't care about it at all, good luck.

**Comment by hayleyxyz** [comment]
Where did you find this "clean" installer? Are you sure it's official, and not just made by some random person?

Yeah, I hate bundled software, but you can decline them all and they won't get installed.

**Comment by fade2gray** [comment]
Yeah, sounds like an unofficial installer that's going around that allows you to change the installation path, but when CE is run, it detects the incorrect path and triggers the laugh (something like that).

https://www.reddit.com/r/computerviruses/comments/11ynymn/comment/jftvewc/?utm_source=reddit&utm_medium=web2x&context=3

**Comment by 19e2k67** [comment]
It's a lot like some Chinese software.(bundling shitting software)

**Comment by IntPtrZero** [comment]
> Where did you find this "clean" installer? Are you sure it's official, and not just made by some random person?
> 
> Yeah, I hate bundled software, but you can decline them all and they won't get installed. Try and calm down a bit.

Noop, the "Clean" version is the real installer of CE, which is copied from TEMP dir during the running time of official installer. The problem is--if you directly run the clean insatller by double click, the "Joke"(at least Dark Byte thinks it's only a fucking joke) lua scripts will be copied and cuasing the creepy laughting sound. You should run it by CMD and add two commands so it work normal. The problem is -- how can i trust this program after these things? So i won't use it anymore.

**Comment by IntPtrZero** [comment]
> Yeah, sounds like an unofficial installer that's going around that allows you to change the installation path, but when CE is run, it detects the incorrect path and triggers the laugh (something like that).
> 
> https://www.reddit.com/r/computerviruses/comments/11ynymn/comment/jftvewc/?utm_source=reddit&utm_medium=web2x&context=3

It's not path, it's startup args.

**Comment by KorewaLidesu** [comment]
@IntPtrZero what is the specific command for it?
